website/docs/r/is_subnet.html.markdown

```diff
@@ -19,8 +19,8 @@ resource "ibm_is_vpc" "example" {
 }
 
 resource "ibm_is_vpc_routing_table" "example" {
-  name   = "example-routing-table"
-  vpc    = ibm_is_vpc.example.id
+  name = "example-routing-table"
+  vpc  = data.ibm_is_vpc.example.id
```

Comment:
This is pointing to above resource right, not the data source!
Please check the interpolation of the `vpc` id